packages/react-components/src/types/Attachment.ts

```diff
@@ -44,48 +83,69 @@ export interface AttachmentMenuAction {
 
 /**
  * @beta
- * A attachment download error returned via a {@link FileDownloadHandler}.
- * This error message is used to render an error message in the UI.
  */
-export interface FileDownloadError {
-  /** The error message to display in the UI */
-  errorMessage: string;
+export interface AttachmentUploadOptions {
+  /**
+   * A string containing the comma separated list of accepted file types.
+   * Similar to the `accept` attribute of the `<input type="file" />` element.
+   * Accepts any type of file if not specified.
+   * @beta
+   */
+  acceptedMimeTypes?: string[];
```

Comment:
Review was to change this to `MediaType` IIRC?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types

Reply:
thanks updated